lib/super_digit.rb

```diff
@@ -1,15 +1,37 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(logn)
+# Space Complexity - O(n) - system stack
 def super_digit(n)
```

Comment:
Since summing is O(log n) and you're doing recursion log n times.  This solution is O( (log n)^2 )